docs/guide/pieces.md

```diff
@@ -1,396 +1,85 @@
----
-title: "Pieces"
----
-
 # Pieces
-What is a Piece?
 
-As experienced A2 developers know, a piece type module defines structure and behavior for a single type of content, such as blog posts, products, or events. If you have hundreds of documents of the same type, that's a piece. If you want to let the user view them in a certain sorting order, that's a piece. If the same item might be appropriate to display on many different pages, that's a piece. Blog posts, events and products are all excellent examples. Less obvious examples include "category" or "tag" content types that serve to create a taxonomy for other pieces.
+"Piece types" are classifications for standalone, structured content. Each database document created with a piece type module is known as a "piece". The piece type defines how pieces are structured, how end users interact with them, and how they might interact with other content. Blog posts, events, and people profiles are all common examples of piece types.
```

Comment:
```suggestion
"Piece types" are classifications for structured content. Events, products, and author profiles are all common examples of piece types. Individual records of a piece type are known as "pieces".
```

**Each database document created with a piece type module is known as a "piece".**  -- this feels like it could have value, but not here. We don't need to talk directly about the database, this is implied. It might be beneficial to have a section on 'how things are stored' in the db, for developers who wish to know.

**The piece type defines how pieces are structured, how end users interact with them, and how they might interact with other content** -- I think this is implied, so I suggest removal. We know that a piece is 'structured content', so that part isn't necessary. A CMS is about creating content that end users can interact with, so I think thats also implied. 

Removed 'blog posts' from this because its just a bit or difficult to understand. Later, I think we use the example of "articles in a blog" to define 'pieces', which is more clear.

Reply:
I don't think we should consider so much to be obvious to a new developer. In some systems the content types only really define the fields available. In Apostrophe we make much more than that available to configure, so I suggest the more complete picture.